stuff/pipelines/rabbitmq_connection.py

```diff
@@ -19,8 +19,16 @@ def connect(
     username = args.rabbitmq_username
     password = args.rabbitmq_password
     credentials = pika.PlainCredentials(username, password)
-    parameters = pika.ConnectionParameters('rabbitmq', 5672, '/',
-                                           credentials, heartbeat=600)
+    parameters = pika.ConnectionParameters('rabbitmq',
+                                           5672,
+                                           '/',
+                                           credentials,
+                                           heartbeat=600,
+                                           # mypy does not support structural
+                                           # typing yet
+                                           # https://github.com/python/mypy/issues/3186
+                                           # type: ignore:
```

Comment:
esto debería ir en la siguiente línea. (además se me fue un `:` de más, jaja)

```suggestion
                                           # type: ignore
```

Reply:
Intenté ponerla en esa línea, pero me marca el error de la longitud de la línea demasiado larga:
```bash
stuff/pipelines/rabbitmq_connection.py:30:80: [pycodestyle] E501 line too long (91 > 79 characters)
                                               blocked_connection_timeout=300.0  # type: ignore
```

Parece que se puede hacer de otra forma y no marcó error.